StablecoinResearch/DualTokenSim/source/Tokens/algorithmic_stablecoin.py

```
@@ -0,0 +1,68 @@
+        self._peg = peg
+        self._tied = False
+
+    @property
+    def peg(self):
+        """
+        Getter for the peg value.
+        """
+        return self._peg
+
+    @peg.setter
+    def peg(self, value: float):
```

Ruff (0.12.2)

Unused method argument: `value`

(ARG002)